app/services/cert/certs.rb

```diff
@@ -24,10 +24,14 @@ def self.ssl_client_key_filename
       Setting['pulp_client_key']
     end
 
+    def self.backend_ca_cert_file(backend)
+      SETTINGS.dig(:katello, backend, :ca_cert_file) || Setting[:ssl_ca_file]
```

Comment:
I honestly cannot think of a scenario where we would have or want a different CA cert for Candlepin or Pulp given our architecture. @jlsherrill can you think of any reason to not just set the CA cert used to Setting[:ssl_ca_file] and reduce the potential configuration options?

Reply:
I didn't know how breaking we would be if we dropped it immediately and if that's acceptable.

Reply:
unless there is some quirk related with a 3rd party CA i can't think of this should be fine.  I would imagine all sorts of things would not work properly if you started using different CAs for differnet subsystems.

Reply:
There is one case. Pulp 3 can use a different CA than the Smart Proxy. For example, one signed by a publicly trusted CA.